canonical/ubuntu-advantage-client
# docs: add sphinx structure

## Proposed Commit Message
docs: add sphinx structure

Add base sphinx structure to serve our documents as html pages;
The structure is similar to what cloud-init does on their side, but there are still some points that we need to discuss:

1. What should we do to the github specific notation on the documents ? Like notes and warnings
2. Do you want to use a different structure ?
3. Should our developer documentation be included ?

## Test Steps
Run `tox -e docs` and verify that the html pages were generated on `docs/build/html`
Please verify that the content is working as expected. specially the redirects in the pages

## Desired commit type


## Checklist
